src/i18n/en-US.properties

```diff
@@ -37,7 +37,11 @@ text_truncated=This file has been truncated due to size limits. Please download
 
 # Error messages
 # Default preview error message
-error_default=We're sorry, the preview didn't load. This file type may not be supported.
+error_generic=We're sorry, the preview didn't load.
```

Comment:
Thoughts on language for the new messages?

Reply:
From my limited experience refreshing the page seems to fix the issue when this error occurs. If that is usually the case maybe append something about refreshing?

Reply:
My opinion is to leave it as is. If we put the expectation that refreshing could potentially fix the issue, it's another thing that can muddy external bug reports.